s64da_benchmark_toolkit/prepare.py

```diff
@@ -4,15 +4,18 @@
 import shutil
 
 from collections import namedtuple
+from sys import exit
 
 from concurrent.futures import ThreadPoolExecutor, as_completed
 from psycopg2 import ProgrammingError
 from subprocess import Popen, PIPE
 from urllib.parse import urlparse
+import threading
 
 from .dbconn import DBConn
 
 Swarm64DAVersion = namedtuple('Swarm64DAVersion', ['major', 'minor', 'patch'])
+cancelEvent = threading.Event()
```

Comment:
Please move down to the class and make it a member. Also, please use snake case: `cancel_event`.

Reply:
done